app/models/landing_page/block/document_list.rb

```diff
@@ -5,6 +5,12 @@ def full_width?
     end
 
     def items
+      hardcoded_items
+    end
+
+  private
+
+    def hardcoded_items
```

Comment:
I don't like this method name - in this context I expected "hardcoded" to mean "hardcoded in the frontend application", but it actually means "specified explicitly in the block configured in the content item".

This confused me in the next commit, because I thought we were falling back to a hardcoded list, when actually we're falling back to the items in the block.

Maybe `explicitly_configured_items` would be better?